openedx/core/djangoapps/programs/tests/mixins.py

```diff
@@ -125,6 +125,19 @@ class ProgramsDataMixin(object):
         }
     ]
 
+    PROGRAMS_CREDENTIALS_DISPLAY_DATA = [
```

Comment:
This mixin is deprecated. Tests should create and manage their own data. Please use a factory. For example, see https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/programs/tests/factories.py.